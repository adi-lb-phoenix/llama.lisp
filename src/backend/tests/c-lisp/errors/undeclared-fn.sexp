(c-lisp
    (define ((main int))
        (call print 4)))
