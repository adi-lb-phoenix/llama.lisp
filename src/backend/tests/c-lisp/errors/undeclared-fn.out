Error in statement: ['call', 'print', 4]
CodegenError: Call to undeclared function: print
